bciers/libs/components/src/form/fields/NestedArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,85 @@
+import Button from "@mui/material/Button";
+import { ArrayFieldTemplateProps, FieldTemplateProps } from "@rjsf/utils";
+
+function BasicFieldTemplate({ children }: FieldTemplateProps) {
+  return <>{children}</>;
+}
+
+const MinusSVG = () => (
```

Comment:
the wireframe has the 'remove ' option as a trashcan icon. Check with designers for consistency across the app